docs/id-lifecycle-management/identity-issuance/android-registration-client/deploy/android-registration-client-configuration.md

```
@@ -1,186 +1,193 @@
 
 registration-default.properties
 
-## **SBI related configurations:**
+## SBI related configurations
 
 Timeouts in milliseconds set during any http calls to SBI.
 
-mosip.registration.mdm.connection.timeout=10000\
-mosip.registration.mdm.RCAPTURE.connection.timeout=40000\
-mosip.registration.mdm.MOSIPDINFO.connection.timeout=5000\
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)